website/src/codegen/jda/main.java.ejs

```diff
@@ -0,0 +1,30 @@
+import net.dv8tion.jda.api.JDABuilder;
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+import net.dv8tion.jda.api.requests.GatewayIntent;
+
+import java.util.EnumSet;
+import java.util.List;
+
+public class Main {
+  private static class MessageReceiveListener extends ListenerAdapter {
+    @Override
+    public void onMessageReceived(MessageReceivedEvent event) {
+      if (event.getAuthor().isBot()) return;
+
+      List<MessageTopLevelComponent> components = List.of(
```

Comment:
This isn't compatible with Java 8, which JDA targets, please check for other usages.